ush/python/pygfs/utils/marine_da_utils.py

```diff
@@ -0,0 +1,105 @@
+import f90nml
+import os
+from logging import getLogger
+import xarray as xr
+
+from wxflow import FileHandler, logit, WorkflowException, AttrDict, parse_j2yaml, jinja
+
+logger = getLogger(__name__.split('.')[-1])
+
+
+@logit(logger)
+def run(exec_cmd):
+    """Run the executable command
+    """
+    logger.info(f"Executing {exec_cmd}")
+    try:
+        logger.debug(f"Executing {exec_cmd}")
+        exec_cmd()
+    except OSError:
+        raise OSError(f"Failed to execute {exec_cmd}")
+    except Exception:
+        raise WorkflowException(f"An error occured during execution of {exec_cmd}")
+
+    pass
+
+
+@logit(logger)
+def link_executable(task_config, exe_name: str) -> None:
+    """Link the executable to the DATA directory
+    """
+    logger.info(f"Link executable {exe_name}")
+    logger.warn("Linking is not permitted per EE2.")
```

Comment:
```suggestion
    logger.warn("WARNING: Linking is not permitted per EE2.")
```
EE2 suggested way to warn.